bmi_config_files/README.md

```diff
@@ -0,0 +1,46 @@
+# BMI Configuration
+BMI requires a configuration file for each model. The LSTM configuration files contains key value pairs that are used by the BMI to run the model. Below are examples and descriptions of each of such keys, and what type of values are associated.
+
+## Meta data
+These key value pairs contain meta data that is not necessary to run the model, but can be useful to make sure that the model is running as expected.  
+- `time_step: '1 hour'` This can be used by the framework to ensure that the model is called at the right time step.
```

Comment:
Is the model actually using this time step information? We have the time step defined in the BMI spec as well.

Reply:
No.  Timestep info is only being pulled from,
```
    _att_map = {
        'model_name':         'LSTM for Next Generation NWM',
        'version':            '1.0',
        'author_name':        'Jonathan Martin Frame',
        'grid_type':          'scalar', 
        'time_step_size':      1,       
        #'time_step_type':     'donno', #unused  
        #'step_method':        'none',  #unused
        #'time_units':         '1 hour' #NJF Have to drop the 1 for NGEN to recognize the unit
        'time_units':         'hour' } 
```
In fact if it was being used and was something other than `1 hour` then, to your previous comment, this hard-coded line would be problematic:
```
        # ----------- The output is area normalized, this is needed to un-normalize it
        #                         mm->m                             km2 -> m2          hour->s    
        self.output_factor_cms =  (1/1000) * (self.cfg_bmi['area_sqkm'] * 1000*1000) * (1/3600)
```

Reply:
also - `time_step` would need to be separated into `time_step_size` and `time_units` per #NJF comment anyways...